tests/integration/campaigns/post/CreateCampaign.spec.ts

```diff
@@ -38,11 +38,12 @@ describe('When a campaign is created', () => {
 
             const campaignPayload = CampaignDomainDataFaker.mocks.createCampaignMock;
 
+            campaignPayload.visibility = 'visible';
```

Comment:
Por conta do serializer o teste está passando somente se o visibility for 'visible' se colocar hidden ele não passa, o createCampaignMock gera aleatório entre o enum.